assets/js/blocks/rating-filter/test/block.tsx

```diff
@@ -258,7 +257,9 @@ describe( 'Filter by Rating block', () => {
 			);
 
 			if ( removeRating4Button ) {
-				userEvent.click( removeRating4Button );
+				act( () => {
+					userEvent.click( removeRating4Button );
+				} );
```

Comment:
I have no idea why these tests suddenly stopped passing, so I needed to add `atc()` here.